content/windows.md

```diff
@@ -430,43 +221,84 @@ The next page will ask what type of installation you want to perform. Since ther
 
 ![Windows installation types](/images/dual-booting/windows-installation-type.jpg)
 
-Finally, the installer will ask you to select where you want to install to. In this scenario, we are installing Windows to a pre-made partition on the shared drive (if you're doing something else, see [Planning the Installation](#planning-the-installation)).
+Finally, the installer will ask you to select where you want to install to. Please follow the instructions for the configuration you chose under [Dual-Booting Windows](#dual-booting-windows-with-linux)
+
+Installing Windows to a **Dedicated Drive**
+
+1. Select the drive you would like to install Windows to.
```

Comment:
"Select the drive for Windows installation."

or maybe,

"Select the drive target for Windows installation."